jenkins/bootstrap.py

```diff
@@ -343,6 +343,7 @@ def finish(gsutil, paths, success, artifacts, build, version, repo):
 
     if os.path.isdir(artifacts) and any(f for _, _, f in os.walk(artifacts)):
         try:
+            call(['sudo', 'chmod', 'a+rX', '-R', artifacts])
```

Comment:
I'm not super excited about repeating this 300 times, so I moved this to `dockerized-e2e-runner.sh`.